### DISCUSSION TITLE
Connection Failure when trying to scan the QR code from rainbow wallet on phone

### DISCUSSION THREAD

**Discussion by wahabshah21** [rainbow-me/rainbowkit#2061] [Q&A]
So when I try to scan the QR code on the rainbow kit modal in my dapp (I am using rainbow phone wallet on android to scan). So it prompts to connect but once I click on the connect button it gives the following error.

"Connection Failed, There was an issue with the namespaces requested by the dapp. Please try again or contact rainbow and/or dapp support team.
No accounts found in approved namespace".

What could be the issue do I have to put something in my config? Same happens with walletConnect QR code in the rainbowkit modal

**Comment by 0xmkh** [comment]
@wahabshah21 Mhmm, are you using your own `projectId` ? We've had a similar issue with WalletConnect here https://github.com/rainbow-me/rainbowkit/issues/2009. Can you make sure you have the right `projectId` and configure it properly in the WalletConnect dashboard ?

If that doesn't work feel free to share a small reproducible example and we'll take a look.
